web-services/handler.js

```diff
@@ -8,9 +8,45 @@ module.exports.ping = async (event, context) => {
       input: event,
     }),
   };
+};
+//--------------------------------------------------
+//This function uses axios to get Trello API boards
+//--------------------------------------------------
+
+const axios = require('axios');
 
+const getData = async () => {
+  try {
+    return await axios.get('https://api.trello.com/1/members/me/boards?', {
+        params: {
+            key: '96e992b35b800ab9db9dc719439519f0',
+            token: '3c50380995f42d4fd2818dc2cdb95aae20c98c1a9231ea037df806ccd4f56898',
```

Comment:
This should not be checked into source control as it is a sensitive piece of information. Instead, look into using environment variables in the serverless.yml.

Reply:
I fixed this issue. I put secret info in environment variables inside a separate file. This file will not included when I push branch to gitHub. It worked well at local. I will test ignore file when I push updated branch